template/koa-rest-api/src/app.test.ts

```diff
@@ -11,10 +11,26 @@ describe('app', () => {
   it('has a happy health check', () => agent.get('/health').expect(200, ''));
 
   it('has a reachable smoke test', () =>
-    agent.get('/smoke').expect(({ status }) => status !== 404));
+    agent.get('/smoke').expect(({ status }) => {
+      if (status === 404) {
+        throw new Error('Unreachable smoke test');
+      }
+    }));
```

Comment:
Thanks for the fixes Luke!

Thoughts on whether this may be cleaner:

```suggestion
  it('has a reachable smoke test', async () => {
    const response = await agent.get('/smoke');

    expect(response.status).not.toBe(404);
  });
```